### ISSUE TITLE
Connected Intel AMT is always 0

### ISSUE THREAD

**Issue by vitko-bg** [Ylianst/MeshCentral#6549]
**Describe the bug**
Have some devices added to intel ATM only, but on on my server tab shows always 0, even when intel AMT session is connected.

**To Reproduce**
Add intel AMT device, check the counter, will be 0
Open separate tab in browser to see the counter, and try to connect to AMT device, once connected, still shows 0. BTW, is this counter shows AMT devices in the group, or currently live connected AMT devices?

**Expected behavior**
To show amount of AMT devices in groups, at least it was like this in previous versions, not sure which ones.

**Screenshots**
![image](https://github.com/user-attachments/assets/ee460f24-c8d1-4670-9085-e7d37eacfe14)

![image](https://github.com/user-attachments/assets/80e9bf7a-551a-4326-9935-52a4a11d5c77)

**Server Software (please complete the following information):**
 - OS: Debian 12
 - Virtualization: none
 - Network: Hybrid (LAN + WAN) mode, nginx - reverse proxy
 - Version: 1.1.33
 - Node: v22.11.0

**Client Device (please complete the following information):**
 - Device: PC
 - OS: AMT
 - Network: Local to Meshcentral
 - Browser: All browsers
 - MeshCentralRouter Version: n/a

**Remote Device (please complete the following information):**
 - Device: any
 - OS: any
 - Network: Local to Meshcentral and Remote over WAN

**Comment by vitko-bg** [comment]
As an addition - have a feeling it all happened after version update related in #6386

**Comment by si458** [comment]
sounds like a very daft one, have you simple tried restarting meshcentral? 
![off and on](https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExcHhqOWo0OTR5cmp1Nzd6dnh4azMyNHN3cXF3OWwwZHJwdzEwcjQ5bSZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/F7yLXA5fJ5sLC/giphy.gif)
after checking the source code `ConnectedIntelAMT` gets retrieved from `parent.parent.mpsserver.ciraConnections`
so its always a number and `ciraConnections` just increases every time the is a cira connection?

**Comment by si458** [comment]
is this still an issue?
the `Connected Intel AMT` value is
how many devices are currently connected to your meshcentral server via CIRA or AMT

**Comment by vitko-bg** [comment]
yes, this is still an issue, as with latest 1.1.43 value is still 0, however there are AMT devices connected to the meshcentral server

**Comment by si458** [comment]
@vitko-bg when you say you have AMT devices connected, how are they connected?

are they connected using AMT or AMT CIRA?

they are 2 different types of things when it comes to connections

the `Connected Intel® AMT` is actually the number of CIRA connected devices,
NOT the number of AMT connected devices (either CIRA or AMT or even overall !)

for example i have a group which only has an AMT device in but NOT setup to use CIRA
where as i have another group which has mixed agent and amt devices, but another AMT device which IS setup with CIRA

so my total would be 1 and not 2

**Comment by vitko-bg** [comment]
Well, they are connected the following way:
1. create an group Intel AMT only
2. add device via interface:

![Image](https://github.com/user-attachments/assets/f01593af-c4df-4cfb-a72f-0779e39e4a0c)

by adding host name, user name, password, etc.

Client device in group with agent connected device looks like this:

![Image](https://github.com/user-attachments/assets/1706fd07-090e-4f01-bd46-890f2e44a8e3)

Client device in AMT Group connected device look like this:

![Image](https://github.com/user-attachments/assets/8b852ca9-a1e1-4317-8e3d-bf52ea5142c5)

I am really sorry, but when you mention CIRA connection in my understanding it is Client Initiated connection, which I can confirm no one is initiating connection from client side, it is only connections initiated from MC to the AMT device. 

Here is some more info:

![Image](https://github.com/user-attachments/assets/0bf3e3c2-a632-4407-8670-1b04d45d9c57)

**Comment by vitko-bg** [comment]
Cool, understood. Thanks @si458 ! Will close the issue, as look like this is not an issue ;)

**Comment by si458** [comment]
No it's OK I'm looking into it now for you so have reopened it!
I'll sort out the ui to make it look better to understand!

**Comment by si458** [comment]
much nicer! all done https://github.com/Ylianst/MeshCentral/commit/36b5af7ad4be53f1c1897f8e20952b197d73c09f off to bed
<img width="1509" alt="Image" src="https://github.com/user-attachments/assets/73165339-6a87-4424-8e05-5408b8e8842c" />
<img width="779" alt="Image" src="https://github.com/user-attachments/assets/6214bfc0-248c-482d-8eef-873381e580d0" />

```
{
	"action": "serverstats",
	"totalmem": 16732725248,
	"freemem": 12512284672,
	"cpuavg": [
		0.47,
		0.91,
		0.79
	],
	"availablemem": 12512284672,
	"values": {
		"ServerState": {
			"UserAccounts": 27,
			"DeviceGroups": 28,
			"AgentSessions": 19,
			"ConnectedUsers": 1,
			"UsersSessions": 2,
			"RelaySessions": 0,
			"RelayCount": 0,
			"ConnectedIntelAMTCira": 1,
			"ConnectedIntelAMT": 1
		}
	}
}
```
```
{
  "time": "2025-03-25T19:26:16.101Z",
  "expire": "2025-03-26T03:26:16.101Z",
  "mem": {
    "rss": 266014720,
    "heapTotal": 89042944,
    "heapUsed": 77873496,
    "external": 35210956,
    "arrayBuffers": 31774325
  },
  "conn": {
    "ca": 19,
    "cu": 1,
    "us": 1,
    "rs": 0,
    "amc": 1,
    "am": 1
  },
  "traffic": {
    "httpRequestCount": 82,
    "httpWebSocketCount": 2,
    "httpIn": 58094,
    "httpOut": 1776431,
    "relayCount": {},
    "relayIn": {},
    "relayOut": {},
    "localRelayCount": {},
    "localRelayIn": {},
    "localRelayOut": {},
    "AgentCtrlIn": 2241,
    "AgentCtrlOut": 239,
    "LMSIn": 0,
    "LMSOut": 0,
    "CIRAIn": 50,
    "CIRAOut": 50,
    "time": 1742930776101,
    "delta": 300000
  }
}
```
```
# HELP meshcentral_connectedintelamt Connected Intel AMT
# TYPE meshcentral_connectedintelamt gauge
meshcentral_connectedintelamt 1

# HELP meshcentral_connectedintelamtcira Connected Intel AMT CIRA
# TYPE meshcentral_connectedintelamtcira gauge
meshcentral_connectedintelamtcira 1

# HELP meshcentral_useraccounts User Accounts
# TYPE meshcentral_useraccounts gauge
meshcentral_useraccounts 27
```
